<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>936</width>
    <height>695</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Vintel</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QSplitter" name="splitter">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="frameShape">
       <enum>QFrame::Panel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="opaqueResize">
       <bool>true</bool>
      </property>
      <property name="handleWidth">
       <number>6</number>
      </property>
      <widget class="QWidget" name="mapwidget" native="true">
       <layout class="QGridLayout" name="gridLayout_3">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <property name="spacing">
         <number>0</number>
        </property>
        <item row="0" column="0">
         <layout class="QVBoxLayout" name="verticalLayout">
          <property name="spacing">
           <number>0</number>
          </property>
          <item>
           <widget class="QWidget" name="mapbuttonwidget" native="true">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>24</height>
             </size>
            </property>
            <property name="baseSize">
             <size>
              <width>1024</width>
              <height>0</height>
             </size>
            </property>
            <layout class="QGridLayout" name="gridLayout_5">
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item row="0" column="0">
              <layout class="QHBoxLayout" name="horizontalLayout_2">
               <item>
                <widget class="QPushButton" name="frameButton">
                 <property name="maximumSize">
                  <size>
                   <width>16777215</width>
                   <height>19</height>
                  </size>
                 </property>
                 <property name="font">
                  <font>
                   <pointsize>8</pointsize>
                  </font>
                 </property>
                 <property name="text">
                  <string>Restore Frame</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="soundButton">
                 <property name="maximumSize">
                  <size>
                   <width>16777215</width>
                   <height>19</height>
                  </size>
                 </property>
                 <property name="font">
                  <font>
                   <pointsize>8</pointsize>
                  </font>
                 </property>
                 <property name="text">
                  <string>Sound</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="zoomInButton">
                 <property name="maximumSize">
                  <size>
                   <width>16777215</width>
                   <height>19</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>+</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="zoomOutButton">
                 <property name="maximumSize">
                  <size>
                   <width>16777215</width>
                   <height>19</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>-</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="jumpbridgesButton">
                 <property name="maximumSize">
                  <size>
                   <width>16777215</width>
                   <height>19</height>
                  </size>
                 </property>
                 <property name="font">
                  <font>
                   <pointsize>8</pointsize>
                  </font>
                 </property>
                 <property name="text">
                  <string>JumpBridges</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="statisticsButton">
                 <property name="maximumSize">
                  <size>
                   <width>16777215</width>
                   <height>19</height>
                  </size>
                 </property>
                 <property name="font">
                  <font>
                   <pointsize>8</pointsize>
                  </font>
                 </property>
                 <property name="text">
                  <string>Statistics</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QWebView" name="map">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="baseSize">
             <size>
              <width>1050</width>
              <height>790</height>
             </size>
            </property>
            <property name="contextMenuPolicy">
             <enum>Qt::DefaultContextMenu</enum>
            </property>
            <property name="layoutDirection">
             <enum>Qt::LeftToRight</enum>
            </property>
            <property name="url">
             <url>
              <string>about:blank</string>
             </url>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QGroupBox" name="chatbox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="baseSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="title">
        <string>All Intel</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_6">
        <property name="leftMargin">
         <number>5</number>
        </property>
        <property name="topMargin">
         <number>5</number>
        </property>
        <property name="rightMargin">
         <number>5</number>
        </property>
        <property name="bottomMargin">
         <number>5</number>
        </property>
        <property name="horizontalSpacing">
         <number>5</number>
        </property>
        <item row="1" column="0">
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QPushButton" name="chatLargeButton">
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>15</height>
               </size>
              </property>
              <property name="text">
               <string>+</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="chatSmallButton">
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>15</height>
               </size>
              </property>
              <property name="text">
               <string>-</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QListWidget" name="chatListWidget">
            <property name="horizontalScrollBarPolicy">
             <enum>Qt::ScrollBarAsNeeded</enum>
            </property>
            <property name="editTriggers">
             <set>QAbstractItemView::NoEditTriggers</set>
            </property>
            <property name="horizontalScrollMode">
             <enum>QAbstractItemView::ScrollPerPixel</enum>
            </property>
            <property name="resizeMode">
             <enum>QListView::Adjust</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>936</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Menu</string>
    </property>
    <widget class="QMenu" name="menuTransparency">
     <property name="title">
      <string>Transparency</string>
     </property>
    </widget>
    <addaction name="alwaysOnTopAction"/>
    <addaction name="menuTransparency"/>
    <addaction name="framelessWindowAction"/>
    <addaction name="floatingOverviewAction"/>
    <addaction name="separator"/>
    <addaction name="kosClipboardActiveAction"/>
    <addaction name="separator"/>
    <addaction name="infoAction"/>
    <addaction name="quitAction"/>
   </widget>
   <widget class="QMenu" name="menuChat">
    <property name="title">
     <string>Chat</string>
    </property>
    <addaction name="showChatAction"/>
    <addaction name="separator"/>
    <addaction name="chooseChatRoomsAction"/>
    <addaction name="showChatAvatarsAction"/>
   </widget>
   <widget class="QMenu" name="menuSound">
    <property name="title">
     <string>Sound</string>
    </property>
    <addaction name="activateSoundAction"/>
    <addaction name="soundSetupAction"/>
    <addaction name="useSpokenNotificationsAction"/>
   </widget>
   <widget class="QMenu" name="menuRegion">
    <property name="title">
     <string>Map</string>
    </property>
    <addaction name="chooseRegionAction"/>
    <addaction name="jumpbridgeDataAction"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menuChat"/>
   <addaction name="menuSound"/>
   <addaction name="menuRegion"/>
  </widget>
  <action name="infoAction">
   <property name="text">
    <string>Info</string>
   </property>
  </action>
  <action name="alwaysOnTopAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Always on top</string>
   </property>
  </action>
  <action name="opac100">
   <property name="text">
    <string>Opacity 100%</string>
   </property>
  </action>
  <action name="opac80">
   <property name="text">
    <string>Opacity 80%</string>
   </property>
  </action>
  <action name="opac60">
   <property name="text">
    <string>Opacity 60%</string>
   </property>
  </action>
  <action name="opac40">
   <property name="text">
    <string>Opacity 40%</string>
   </property>
  </action>
  <action name="opac20">
   <property name="text">
    <string>Opacity 20%</string>
   </property>
  </action>
  <action name="chooseChatRoomsAction">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Choose Chatrooms...</string>
   </property>
  </action>
  <action name="showChatAvatarsAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show Chat Avatars</string>
   </property>
  </action>
  <action name="kosClipboardActiveAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>KOS-Check Clipboard on Foreground</string>
   </property>
   <property name="toolTip">
    <string>Activate Clipboard KOS-Check</string>
   </property>
  </action>
  <action name="showChatAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show Chat</string>
   </property>
  </action>
  <action name="activateSoundAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Activate Sound</string>
   </property>
  </action>
  <action name="framelessWindowAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Frameless Main Window</string>
   </property>
   <property name="toolTip">
    <string>Frameless Main Window</string>
   </property>
  </action>
  <action name="quitAction">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="chooseRegionAction">
   <property name="text">
    <string>Choose Region...</string>
   </property>
  </action>
  <action name="soundSetupAction">
   <property name="text">
    <string>Sound Setup...</string>
   </property>
  </action>
  <action name="jumpbridgeDataAction">
   <property name="text">
    <string>Jumpbridge Data...</string>
   </property>
  </action>
  <action name="useSpokenNotificationsAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Spoken Notifications</string>
   </property>
   <property name="toolTip">
    <string>Spoken KOS results</string>
   </property>
  </action>
  <action name="floatingOverviewAction">
   <property name="text">
    <string>Show Mini Overview...</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QWebView</class>
   <extends>QWidget</extends>
   <header>QtWebKitWidgets/QWebView</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
